res/schema.xml

```diff
@@ -489,4 +489,29 @@ METADATA
       ALTER TABLE library ADD COLUMN color INTEGER;
     </sql>
   </revision>
+  <revision version="32" min_compatible="3">
+    <description>
+      Convert the PredefinedColor ID to the actual RGB value.
+    </description>
+    <sql>
+      <!-- No Color becomes Orange (not in default palette) -->
+      UPDATE cues SET color=0xFF8C00 WHERE color=0;
+      <!-- Red -->
+      UPDATE cues SET color=0xC50A08 WHERE color=1;
+      <!-- Green -->
+      UPDATE cues SET color=0x32BE44 WHERE color=2;
+      <!-- Blue -->
+      UPDATE cues SET color=0x0044FF WHERE color=3;
+      <!-- Yellow -->
+      UPDATE cues SET color=0xF8D200 WHERE color=4;
+      <!-- Celeste -->
```

Comment:
"celeste" isn't a color word in english.  This looks like "Cyan"

Reply:
Ok, I took this from: https://mixxx.org/wiki/lib/exe/detail.php/cuecolorsreference.svg?id=hotcue_colors

Reply:
hah! apparently it is english but I've never heard it before.  Fine to leave as is I guess